front/src/styles/scss/applications/stdcm/_home.scss

```
@@ -294,6 +298,35 @@
         line-height: 24px;
         text-align: center;
       }
+
+      .validation-message {
+        width: 450px;
+        display: flex;
+        font-weight: 400;
+        letter-spacing: 0px;
+        text-align: center;
+        justify-content: center;
+        line-height: 32px;
+        height: 80px;
+        border-radius: 8px;
+        color: rgb(33, 100, 130);
+        background-color: rgb(230, 247, 255);
```

We try to directly use the design system theme from `_variables.scss`
```suggestion
        background-color: var(--info5);
```

You can also use `info60` for the text color, add `success5` (in hex unit) for the success background and use `success60` for the success text

---

done

---

done